### PR TITLE
faq.php: typos etc.

### PR DESCRIPTION
**Pull request by grahamperrin**
Nits, including ABIs (plural of ABI).

Make a separate paragraph from the two sentences about no quarterly for CURRENT.

### CODE REVIEW

File: www/faq.php [hunk 1]
```diff
@@ -941,13 +943,13 @@
    </td></tr>
 
 <tr id="abi">
-<?php echo freshports_PageBannerText('Why isthis particular ABI not listed on the website??'); ?>
+<?php echo freshports_PageBannerText('Why is this particular ABI not listed on the website?'); ?>
 </tr>
 
    <tr><td class="textcontent">
-   <p>FreshPorts covers the ABI published by the project at https://pkg.freebsd.org/index.html</p>
+   <p>FreshPorts covers the ABIs published by the project at https://pkg.freebsd.org/index.html.</p>
```
Comment (grahamperrin) [inline www/faq.php:950]: I assume that the trailing `.` will not be interpreted as part of the URL …

**Comment by dlangille** [comment]
Sorry, I was reading this not realizing it was a merge request, and I did [ed156f9](https://github.com/FreshPorts/freshports/commit/ed156f99d997b4a2064235eaa06e1497abd3e8c5)

File: www/faq.php [hunk 2]
```diff
@@ -941,15 +943,15 @@
...
 
 <tr id="abi">
-<?php echo freshports_PageBannerText('Why isthis particular ABI not listed on the website??'); ?>
+<?php echo freshports_PageBannerText('Why is this particular ABI not listed on the website?'); ?>
 </tr>
 
    <tr><td class="textcontent">
    <p>FreshPorts covers the ABIs published by the project at <a href="https://pkg.freebsd.org/index.html">https://pkg.freebsd.org/index.html</a>.</p>
 
    <p>In general, the project does not build quarterly branches for whatever branch is now CURRENT.</p>
```
Comment (grahamperrin) [inline www/faq.php:952]: This paragraph might be superfluous. 

<https://dev.freshports.org/faq.php#packages> (above) states "… No packages are built for that ABI/package set combination.".  

Whilst the foot of <https://dev.freshports.org/--/package-imports.php?sort=name> does, currently, show six _quarterly_ rows for FreeBSD:15:⋯, those peculiarities are not evident in this section of the FAQ page.

Comment (dlangille) [reply www/faq.php:952]: I'll leave the superfluous.  People often read only one FAQ.

... I think your last sentence asks: If we don't build quarterly for current, why is FreshPorts showing that?

The answer is: it's easier than figuring out what is `current`.

**Comment by dlangille** [comment]
I had some problem when merging, and I hope I resolved them properly.  The changes are now on `dev`.

**Comment by grahamperrin** [comment]
Not quite resolved, now I made a mess of things on my side :-) I'll make another PR when my head is straight.

**Pull request merged**